packages/sync-server/src/sync/CentralSyncManager.ts

```diff
@@ -61,12 +62,12 @@ export class CentralSyncManager {
     };
   }
 
-  async tickTockGlobalClock(): Promise<GlobalClockResult> {
+  async tickTockGlobalClock(models: SyncServerModelRegistry): Promise<GlobalClockResult> {
```

Comment:
```suggestion
  async tickTockGlobalClock(transactingModels: SyncServerModelRegistry): Promise<GlobalClockResult> {
```
I presume that's why we're not using `this.models`